src/site/_includes/codelab.njk

```diff
@@ -30,7 +30,7 @@ permalink: "/{{lang}}/{{page.fileSlug}}/index.html"
             {% Author {
               post: page,
               author: contributors[author],
-              avatar: author,
+              id: author,
```

Comment:
This seems unrelated but was presumably a bug.

Reply:
yeah i mentioned it a bit in the original comment. I noticed codelabs weren't displaying the author portrait.